repos/spack_repo/builtin/packages/texinfo/package.py

```diff
@@ -63,12 +66,23 @@ class Texinfo(AutotoolsPackage, GNUMirrorPackage):
 
     patch("nvhpc.patch", when="%nvhpc")
 
-    @property
-    def build_targets(self):
-        targets = []
-        if self.spec.satisfies("@7.0:"):
-            targets.append(f"CFLAGS={self.compiler.c11_flag}")
-        return targets
+    def configure_args(self):
+        spec = self.spec
+        config_args = []
+
+        if spec.satisfies("+xs"):
+            config_args.append("--enable-perl-xs")
+        else:
+            config_args.append("--disable-perl-xs")
+
+        return config_args
+
+    def flag_handler(self, name, flags):
+        if self.spec.satisfies("@7.0:7.1"):
+            if name == "cflags":
```

Comment:
```suggestion
        if name == "cflags":
            if self.spec.satisfies("@7.0:7.1"):
```

I'd flip the conditionals here to make it easier to add additional flags in the future.

Reply:
Good point, done.